lib/cchain/queryutil/rewards.go

```diff
@@ -15,7 +14,7 @@ import (
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 )
 
-var blocksPerYear = math.LegacyNewDec(365 * 24 * 60 * 60 / 2) // Matches upgrades/magellan::blocksPerYear
+var yearDuration = math.LegacyNewDec(365 * 24 * time.Hour.Milliseconds())
```

Comment:
nit: this isn't "duration",it is number of milliseconds, so suggest `yearMillis`